driver/controller.go

```diff
@@ -162,21 +162,24 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	contentSource := req.GetVolumeContentSource()
+	var snapshot *godo.Snapshot
 	if contentSource != nil && contentSource.GetSnapshot() != nil {
 		snapshotID := contentSource.GetSnapshot().GetSnapshotId()
 		if snapshotID == "" {
 			return nil, status.Error(codes.InvalidArgument, "snapshot ID is empty")
 		}
 
 		// check if the snapshot exist before we continue
-		_, resp, err := d.snapshots.Get(ctx, snapshotID)
+		var resp *godo.Response
+		snapshot, resp, err = d.snapshots.Get(ctx, snapshotID)
 		if err != nil {
 			if resp != nil && resp.StatusCode == http.StatusNotFound {
 				return nil, status.Errorf(codes.NotFound, "snapshot %q does not exist", snapshotID)
 			}
 			return nil, err
 		}
 
+		log.WithField("snapshot_size_giga_bytes", snapshot.SizeGigaBytes).Info("snapshot size giga bytes")
```

Comment:
I suggest we include the field with the next line where we log a related message about the snapshot anyway.